tests/unit/legacyTestConfig.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Let's put this into a subdir (it's possible there will be more of these as the project evolves).  perhaps, call it testConfigs